tests/helper/helper_oc.go

```diff
@@ -202,7 +209,7 @@ func (oc *OcRunner) MaxCPU(componentName string, appName string, project string)
 	return maxMemory
 }
 
-// MinCPU reuturns maximum cpu
+// MinCPU reuturns minimum cpu
```

Comment:
you resolved the `maximum` to `minimum` but missed the `reuturns` :laughing: ?

Reply:
👍